cli/root.go

```diff
@@ -48,50 +44,10 @@ func RootCmd() *cobra.Command {
 	// Disable completion default cmd
 	cmd.CompletionOptions.DisableDefaultCmd = true
 	// Persistent flags
-	cmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.sedge.yaml)")
+	cmd.PersistentFlags().StringVar(&logLevel, "logLevel", "info", "Set Log Level")
```

Comment:
The help message should be:
```
Set Log Level, e.g. info, debug, error, ...
```

Put all the possible values there ^^^

Reply:
solved at 6e38108